charts/wasmcloud-platform/README.md

```diff
@@ -0,0 +1,120 @@
+# wasmcloud-platform
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.2.0](https://img.shields.io/badge/AppVersion-1.2.0-informational?style=flat-square)
+
+[wasmCloud](https://wasmcloud.com/docs/intro) is an open source project from the Cloud Native Computing Foundation (CNCF) that enables teams to build polyglot applications composed of reusable Wasm components and run them—resiliently and efficiently—across any cloud, Kubernetes, datacenter, or edge.
+
+The wasmCloud Platform Helm Chart provides a turnkey solution, for running WebAssembly applications on Kubernetes.
+
+## Prerequisites
+
+*   Kubernetes: `>=1.24.0`
+
+*   Helm: `>=3.8.0`
+
+*   If installing the NATS server - a StatefulSet - the means of provisioing PVCs (e.g. a CSI driver)
+
+## Getting Started
+
+The wasmCloud platform is comprised of three components, a NATS cluster as the backbone of its [lattice](https://wasmcloud.com/docs/concepts/lattice), [Wadm](https://wasmcloud.com/docs/ecosystem/wadm) for WebAssembly applications lifecycle management, and wasmCloud [host](https://wasmcloud.com/docs/concepts/hosts), which can be provided either by wasmcloud-operator - if preferring managed hosts - or wasmcloud-host.
+
+To take full advantage of the chart features, it's best to use wasmcloud-operator, and install the chart in two steps.
+
+### Deploy wasmCloud Platform
+
+```bash
+helm upgrade --install \
+    --create-namespace \
+    wasmcloud-platform \
+    --values https://raw.githubusercontent.com/wasmCloud/wasmcloud/main/charts/wasmcloud-platform/values.yaml \
+    oci://ghcr.io/wasmcloud/charts/wasmcloud-platform \
+    --dependency-update \
+    --set "nats.enabled=true" \
+    --set "wadm.enabled=true" \
+    --set "operator.enabled=true" \
+    --set "host.enabled=false" \
+    --set "hostConfig.enabled=false"
+```
+
+Wait for all components to install and wadm-nats communications to establish:
+
+```bash
+kubectl wait --for=condition=available --timeout=600s deploy -l app.kubernetes.io/name=wadm
+kubectl wait --for=condition=available --timeout=600s deploy -l app.kubernetes.io/name=wasmcloud-operator
```

Comment:
I would add the following before `wadm` and the `wasmcloud-opertor`:
```shell
kubectl rollout status deploy,sts -l app.kubernetes.io/instance=nats
```

Reply:
Wadm installation won't succeed until NATS endpoint is available; that's why I found of redundant to check for NATS status too; if you think it's still beneficial I'll add a wait variant, as NATS cluster takes a bit of time to settle.